README.md

```diff
@@ -30,7 +30,6 @@ Additional tools that use the libraries created by the reconciliations are also
 - `openshift-groups`: Manages OpenShift Groups.
 - `openshift-namespaces`: Manages OpenShift Namespaces.
 - `openshift-network-policies`: Manages OpenShift NetworkPolicies.
-- `openshift-resources-annotate`: Annotates OpenShift Resources so they can be used by the `openshift-resources` integration.
```

Comment:
Any special reason why is this part of this PR?

Reply:
Since this pr would have required changes in this "integration", and it is not used already, i figured a cleanup would be best...